test/gpu/cuda_tests.jl

```diff
@@ -1,73 +1,77 @@
-@testitem "Solving on GPUs" tags=[:cuda] skip=:(using CUDA; !CUDA.functional()) begin
+@testitem "Solving on GPUs" tags=[:cuda] begin
```

Comment:
Note that DI is completely untested on GPUs at the moment, so issues may (will) arise

Reply:
what's the reason for that? Do you not have the buildkite access setup?

Reply:
I haven't gotten around to it yet, CPU AD kept me busy enough ^^ but now that the API has stabilized I can look into it